resources/js/components/MealCardComponent.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="mealCArd">
-        <a class="mealCArdInfo" :href="`/recipe/${+id}`">
+        <a class="mealCArdInfo" :href="`/recipe/${+id}/week=${+week}`">
```

Comment:
@CapFrostyMug А почему тут не так?
```suggestion
        <a class="mealCArdInfo" :href="`/recipe/${+id}/week/${+week}`">
```